web/src/enterprise/campaigns/global/marketing/CampaignsUserMarketingPage.tsx

```diff
@@ -1,14 +1,7 @@
 import React from 'react'
 import { CampaignsMarketing } from './CampaignsMarketing'
 
-export interface CampaignsUserMarketingPageProps {
-    /** When true, the message shown will indicate that the user can just not use it because read-access has not been provided to non site-admins. */
-    enableReadAccess: boolean
-}
-
-export const CampaignsUserMarketingPage: React.FunctionComponent<CampaignsUserMarketingPageProps> = ({
-    enableReadAccess,
-}) => (
+export const CampaignsUserMarketingPage: React.FunctionComponent = () => (
```

Comment:
```suggestion
export const CampaignsUserMarketingPage: React.FunctionComponent<{}> = () => (
```

just to indicate it intentionally doesn't accept anything :)